featuretools/computational_backends/feature_set_calculator.py

```diff
@@ -531,27 +532,27 @@ def _calculate_direct_features(self, features, child_df, df_trie, progress_callb
 
         parent_df = df_trie.get_node([path[0]]).value
         _is_forward, relationship = path[0]
-        merge_var = relationship.child_column.id
+        merge_var = relationship.child_column.name
```

Comment:
more places to use `_parent_column_name` and `_child_column_name` through line 547